plugins/sqlfluff-templater-dbt/test/rules_test.py

```diff
@@ -1,19 +1,18 @@
 """Tests for the standard set of rules."""
-import pytest
 import os
 import os.path
 from pathlib import Path
-
-from sqlfluff.core import Linter
-from sqlfluff.core.config import FluffConfig
-from sqlfluff.utils.testing.rules import assert_rule_raises_violations_in_file
-
 from test.fixtures.dbt.templater import (  # noqa
```

Comment:
`test` here is a first party library, but only in the test section of the codebase. There might need to be a nested config file to mark this as first party so that it's not grouped up here.